server/opts.go

```diff
@@ -1024,6 +1025,8 @@ func (o *Options) processConfigFileLine(k string, v any, errors *[]error, warnin
 			*errors = append(*errors, err)
 			return
 		}
+	case "default_sentinel":
```

Comment:
Probably should add checks if this is no in operator mode and if no auth callout defined, this should error?

Reply:
It does when it processes - same if the JWT is invalid (wouldn't be any different than the client providing the wrong JWT).

Reply:
@derekcollison added a check here
https://github.com/nats-io/nats-server/pull/6577/files#diff-4f7238cd9f97e766d6c31311be7bcc154b4d313678f513bfca50317dff080dafR72-R75